src/controllers/dlgcontrollerlearning.cpp

```diff
@@ -484,7 +484,7 @@ void DlgControllerLearning::loadControl(const ConfigKey& key,
 void DlgControllerLearning::controlPicked(const ConfigKey& control) {
     QString title = m_pControlPickerMenu->controlTitleForConfigKey(control);
     QString description = m_pControlPickerMenu->descriptionForConfigKey(control);
-    loadControl(control, title, description);
+    loadControl(control, title, std::move(description));
```

Comment:
DlgControllerLearning::loadControl() can make use of a refactoring to use const & 
std::move dos not really help her, because the string is not kept in a member variable.